src/utils/chains.ts

```diff
@@ -26,11 +25,3 @@ export const getBlockExplorerLink = (
     return getExplorerLink(address, chain.blockExplorerUriTemplate)
   }
 }
-
-export const getShortName = (chainId: string): string | undefined => {
-  return Object.keys(chains).find((prefix) => chains[prefix] === chainId)
-}
-
-export const getChainId = (shortName: string): string | undefined => {
-  return chains[shortName]
-}
```

Comment:
This logic is still used in a number of places. I think we should keep it/its tests.

Reply:
It's only used in one place (useChainId), inline makes more sense to me.